tests/Doctrine/Tests/ODM/PHPCR/Query/QueryTest.php

```diff
@@ -184,22 +210,18 @@ public function testGetSingleResult_withOneResult()
         $this->assertEquals('ok1', $res);
     }
 
-    /**
-     * @expectedException \Doctrine\ODM\PHPCR\Query\QueryException
-     */
     public function testGetSingleResult_withTwoResults()
     {
+        $this->expectException(QueryException::class);
         $this->phpcrQuery->expects($this->once())
             ->method('execute')
             ->will($this->returnValue(array('ok1', 'ok2')));
         $this->query->getSingleResult(Query::HYDRATE_PHPCR);
```

Comment:
This line might not needed?

Reply:
the ordering is wrong. the line right after expect is setting up the partial mock to run the query.